mock/mock_test.go

```diff
@@ -1613,6 +1613,50 @@ func Test_Arguments_Diff_WithAnythingOfTypeArgument_Failing(t *testing.T) {
 
 }
 
+func Test_Arguments_Diff__WithAnythingOfTypeArgument_NilType(t *testing.T) {
```

Comment:
You have an extra underscore here, please look at test names above to maintain consistency :)